rzo.gemspec

```diff
@@ -0,0 +1,49 @@
+# coding: utf-8
+
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'rzo/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = 'rzo'
+  spec.version       = Rzo.version
+  spec.licenses      = ['Apache 2.0']
+  spec.authors       = ['Garrett Honeycutt']
+  spec.email         = ['garrett@learnpuppet.com']
+
+  spec.summary       = 'rzo - Layered Puppet control repositories in Vagrant'
```

Comment:
`Rizzo (rzo) is a tool for working with Vagrant and layered Puppet control repos`